### PR TITLE
[tf-module:cargohold] Replace subnet IDs input with route table IDs

### PR DESCRIPTION
**Pull request by lucendio**
This removes the 'implicit' data dependency that can cause a race condition
between subnets and route tables. It makes it more clear what the module depends
on and should make it more easy to `depends_on` in the future (TF v0.13).

### CODE REVIEW

File: terraform/modules/aws-cargohold-asset-storage/outputs.tf [hunk 1]
```diff
@@ -26,4 +22,8 @@ output "cargohold_access_key" {
 
 output "cargohold_access_secret" {
   value = aws_iam_access_key.cargohold.secret
-}
\ No newline at end of file
+}
+
+output "talk_to_S3" {
+  value = aws_security_group.talk_to_S3.id
```
Comment (lucendio) [inline terraform/modules/aws-cargohold-asset-storage/outputs.tf:28]: Maybe I'm missing sth here, but I can't find the security group resource this output refers to.

**Review by lucendio — COMMENTED** [review]
Sth is off here. I think, a piece of the puzzle is missing...

File: terraform/modules/aws-cargohold-asset-storage/resources.security_groups.tf [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    from_port   = 443
+    to_port     = 443
+    protocol    = "tcp"
+    cidr_blocks = aws_vpc_endpoint.s3.cidr_blocks
+  }
+
+  tags = {
+    Name = "talk_to_S3"
+  }
+}
```
Comment (lucendio) [inline terraform/modules/aws-cargohold-asset-storage/resources.security_groups.tf:19]: EOF
```suggestion
}

```

File: terraform/modules/aws-cargohold-asset-storage/resources.security_groups.tf [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# a security group for access to the private traffic between kubernetes nodes. should be added to all kubernetes nodes.
```
Comment (lucendio) [inline terraform/modules/aws-cargohold-asset-storage/resources.security_groups.tf:1]:
```suggestion
```

File: terraform/modules/aws-cargohold-asset-storage/resources.security_groups.tf [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+resource "aws_security_group" "talk_to_S3" {
+  name        = "talk_to_S3"
+  description = "hosts that are allowed to talk to S3."
+  vpc_id      = var.vpc_id
+
+  # S3
+  egress {
```
Comment (lucendio) [inline terraform/modules/aws-cargohold-asset-storage/resources.security_groups.tf:8]:
```suggestion

  egress {
```
IMHO redundant to the resource description.

**Review by lucendio — COMMENTED** [review]
Ahh, this makes much more sense. Feel free to approve your own work, because I can't. I'm the author of that PR.

**Review by julialongtin — APPROVED** [review]

**Pull request merged**